### PR TITLE
Mod0

### PR DESCRIPTION
**Pull request by tomtl**
Completing week one.

### CODE REVIEW

File: app/views/users/new.html.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      = bootstrap_form_for(@user, layout: :horizontal, label_col: "col-sm-2", control_col: "col-sm-6") do |f|
+        %header
+          %h1 Register
+        %fieldset
+          = f.email_field :email, label: "Email Address"
+          = f.password_field :password
+          = f.text_field :full_name, label: "Full Name"
+        %fieldset.actions.control-group.col-sm-offset-2
+          .controls
+            %input(type="submit" value="Sign Up" class="btn btn-default")
```
Comment (brandoncc) [inline app/views/users/new.html.haml]: Any specific reason you didn't use the form builder for the submit button?

File: spec/models/video_spec.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  end
+
+  it "returns an empty array for a search with an empty string" do
+    futurama = Video.create(title:"Futurama", description: "Space travel")
+    back_to_future = Video.create(title: "Back to the Future", description: "Time travel")
+    expect(Video.search_by_title("")).to eq([])
+  end 
+end
+
+describe "show videos in category" do
```
Comment (brandoncc) [inline spec/models/video_spec.rb]: Did you forget to implement these?

Comment (tomtl) [inline spec/models/video_spec.rb]: I added them in the category_spec but forgot to remove them from here. Fixed now.

File: app/views/users/new.html.haml (hunk 1 above)
Comment (tomtl) [inline app/views/users/new.html.haml]: No reason, that's just how it was in the solution video. I'm still getting used to haml syntax so it didn't jump out at me when I was doing it. It does make more sense now that it is fixed though. Thanks for picking this up!

**Pull request merged**